src/js/definitions/phone.js

```diff
@@ -22,7 +22,7 @@ function uiSchema(title = 'Phone') {
       pattern: 'Phone numbers must be 10 digits'
     },
     'ui:options': {
-      widgetClassNames: 'home-phone va-input-medium-large',
```

Comment:
The classes across the SCSS and JS files were just renamed. I didn't see this particular class defined anywhere so it may be missing?

Reply:
Yes I don't see this class defined anywhere in our codebase, so I think we can just remove it from here entirely, if it's not actually adding any styling.

Reply:
Minor nit, but I think this class, `usfs-input-medium-large`, can be removed.